src/transformers/pipelines.py

```diff
@@ -1382,7 +1387,14 @@ def __call__(self, *args, **kwargs):
                     return_attention_mask=False,
                     return_tensors=self.framework,
                     truncation=True,
+                    return_special_tokens_mask=True,
+                    return_offsets_mapping=self.tokenizer.is_fast,
                 )
+                if "offset_mapping" in tokens:
+                    offset_mapping = tokens["offset_mapping"].cpu().numpy()[0]
+                    del tokens["offset_mapping"]
+                special_tokens_mask = tokens["special_tokens_mask"].cpu().numpy()[0]
```

Comment:
This seems like it would break if ran from a tensorflow pipeline

Reply:
oh I see that, will fix

Reply:
Alright, thanks for making sure it works!